src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -1156,10 +1180,18 @@
     <value>Tab title</value>
     <comment>Header for a control to determine the title of the tab. This is represented using a text box.</comment>
   </data>
+  <data name="Profile_AnswerbackMessage.Header" xml:space="preserve">
+    <value>ENQ response</value>
+    <comment>{Locked=ENQ} Header for a control to determine the response to the ENQ escape sequence. This is represented using a text box.</comment>
+  </data>
   <data name="Profile_TabTitle.HelpText" xml:space="preserve">
     <value>Replaces the profile name as the title to pass to the shell on startup.</value>
     <comment>A description for what the "tab title" setting does. Presented near "Profile_TabTitle".</comment>
   </data>
+  <data name="Profile_AnswerbackMessage.HelpText" xml:space="preserve">
+    <value>The response that is sent when an ENQ control character is received.</value>
+    <comment>{Locked=ENQ} A description for what the "tab title" setting does. Presented near "Profile_TabTitle".</comment>
+  </data>

```

Comment:
Should this be further up next to the header element?
ENQ is not a single control character but a sequence, right?
It may also be helpful to write that ENQ is the VT answerback feature.

Reply:
Oops. I totally didn't change the comment there haha.

I think I'm actually going to move this one over to #17895 and put it in the Terminal subpage.